product_secondary_unit/views/product_views.xml

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="product_template_form_view" model="ir.ui.view">
+        <field name="name">Product template Secondary Unit</field>
+        <field name="model">product.template</field>
+        <field name="inherit_id" ref="product.product_template_form_view" />
+        <field name="arch" type="xml">
+            <xpath expr="//page[@name='general_information']/group[2]" position="after">
```

Comment:
The only thing I'd change here is to use a relative reference.
And set columns for the group and also for the field as well.
`<xpath
                expr="//page[@name='general_information']/group[last()]"
                position="after"
            >
                <group
                    string="Secondary Unit of Measure"
                    col="4"
                    groups="uom.group_uom"
                >
                    <field name="secondary_uom_ids" colspan="4" nolabel="1">`